package.json

```diff
@@ -32,11 +32,14 @@
     "@typechain/hardhat": "^2.3.0",
     "@types/bunyan": "^1.8.7",
     "@types/chai": "^4.2.21",
+    "@types/chai-as-promised": "^7.1.4",
     "@types/mocha": "^9.0.0",
     "@types/node": "^16.7.10",
     "@typescript-eslint/eslint-plugin": "^5.4.0",
     "@typescript-eslint/parser": "^5.4.0",
+    "bitdao-token-contract": "windranger-io/bit-token-contract#v1.0.0",
     "chai": "^4.3.4",
+    "chai-as-promised": "^7.1.1",
```

Comment:
question: where do you need `chai-as-promised`?

Reply:
I couldn't use .to.be.rejected without it.